sympy/polys/rootoftools.py

```diff
@@ -394,6 +395,25 @@ def free_symbols(self):
         # symbols
         return set()
 
+    def _eval_power(self, other):
+        if not isinstance(other, (Integer, int)):
+            return None
+        if other == 0:
+            return S.One
+        if other == 1:
+            return self
+        if abs(other) < self.poly.degree():
```

Comment:
Why can't you not do `if 0 < other < self.poly.degree():`?
There are some instances where negative powers less than degree can be inverted